css/style.css

```diff
@@ -0,0 +1,335 @@
+
+*{
```

Comment:
Folder structure is not correct here, try to gather similar information into a folder for e.g. constants, helpers etc

Reply:
done